cypress/component/Input.cy.jsx

```diff
@@ -0,0 +1,98 @@
+import Input from "@/components/admin/Input";
+import { useState } from "react";
+
+describe("Input", () => {
+  it("Placeholder", () => {
+    const Parent = () => {
+      const [user, setUser] = useState({
+        name: "",
+      });
+      return (
+        <Input
+          name="name"
+          type="text"
+          title="title"
+          placeholder="Enter your name"
+          value={user.name}
+          user={user}
+          setUser={setUser}
+        />
+      );
+    };
+
+    cy.mount(<Parent />);
+    cy.get("p").should("have.text", "Name");
```

Comment:
look at reference code carefully. We are using "data-cy-" to grab this data specifically for cypress testing. You have to add this label in the main component/Input file. Again check out the reference code for the other input test.